Description: Disable SystemUtilsTest#testGetUserHome as it depends on
 $HOME (which is not available on buildd).
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2012-05-27
Forwarded: not-needed
--- a/src/test/java/org/apache/commons/lang3/SystemUtilsTest.java
+++ b/src/test/java/org/apache/commons/lang3/SystemUtilsTest.java
@@ -259,6 +259,7 @@
      * Assumes no security manager exists.
      */
     @Test
+    @org.junit.Ignore
     public void testGetUserHome() {
         final File dir = SystemUtils.getUserHome();
         assertNotNull(dir);
